api/orders.js

```diff
@@ -155,10 +152,8 @@ function getOrders(request, response, next) {
  *  @query
  *  @param {String "true"|"false"} request.query.validated    - used to force request to wait until rippled has finished validating the submitted transaction
  *
- *  @param {Express.js Response} response
```

Comment:
Maybe keep the jsdoc block updated

```javascript
* @callback requestCallback
* @param {Error} [err]
* @param {Object} responseObject

* @param {requestCallback} cb
```


(http://usejsdoc.org/tags-param.html)